src/cli/commands/clean.js

```diff
@@ -120,7 +120,9 @@ export async function run(
   args: Array<string>,
 ): Promise<void> {
   reporter.step(1, 2, reporter.lang('cleanCreatingFile', CLEAN_FILENAME));
-  await fs.writeFile(path.join(config.cwd, CLEAN_FILENAME), '\n', {flag: 'wx'});
+  try {
+      await fs.writeFile(path.join(config.cwd, CLEAN_FILENAME), '\n', {flag: 'wx'});
+  } catch (e) {}
```

Comment:
Catch is suppressed, because this error does not reflect a problem. The command will execute just fine. As checking with fs.exists is deprecated, this is a valid solution, I guess.